### ISSUE TITLE
NLL Loss Down and Up During Training

### ISSUE THREAD

**Issue by jytime** [andreas128/SRFlow#23]
Hi authors,

Thanks for sharing your work. I am trying to extend this model into other datasets. During training, the nll loss first decreases from 1 to -4.5 in the first 3k steps, and then goes back to -0.5 in the later 3k steps. A 10 times larger or smaller learning rate does not prevent this phenomenon. 

Have you observed similar results, or any hints available?

Thanks for your help

**Comment by T5eng** [comment]
I also encountered similar issue when fine-tuning the pretrained model.

It is also wired to see a negative loss value since we are try minimizing this objective.

Wish getting reply from author.

**Comment by jytime** [comment]
@T5eng yeah It's normal. Just go with it
